Kconfig.nrf

```diff
@@ -11,9 +11,23 @@ menu "Nordic nRF Connect"
 config BOARD_THINGY91_NRF9160NS
 	bool
 
-# nRF Connect SDK needs a larger default stacks for running certain tests.
+# Handle default to ensure that BT only will disable ENTROPY_CC3XX,
+# while multi protocol does not.
+config ENTROPY_CC3XX
+	default y if ZIGBEE || OPENTHREAD_NRF_SECURITY
+	default n if BT
```

Comment:
No need to have `default n if BT`. The default value is `n`

Reply:
No. The construct here (due to the order of parameters) will do default n for BT only if Zigbee or OpenThread is not set. This is intentional

Reply:
I see. Why not add `if ZIGBEE || OPENTHREAD_NRF_SECURITY` at the place where `config ENTROPY_CC3XX` is declared? That removes duplication

Reply:
This has been discussed with one of the code-owners. The addition of subsystem knowledge on the level of the driver was rejected as a sub-optimal solution. Also the statement would look very hairy

something like this:
default y if !(BT & !(ZIGBEE || OPENTHREAD_NRF_SECURITY || SPM || BUILD_WITH_TFM))

It was decided that this would be better suited in the Kconfig.nrf scope

Reply:
I don't think it has to be hairy, to me it seems like this would do the exact same thing
```
drivers/entropy/Kconfig
@@ -10,7 +10,7 @@ config ENTROPY_CC3XX
-       default y
+       default y if !BT
subsys/zigbee/Kconfig
@@ -16,6 +16,7 @@ menuconfig ZIGBEE
+       imply ENTROPY_CC3XX
```
And adding imply ENTROPY_CC3XX the other places you mentioned, assuming it is imply and not select.

That doesn't look worse than splitting Kconfig definition into two, atleast using imply instead seems to handle "avoid subsystem knowledge in other places" argument.

Reply:
I will prefer that @anangl or code-owners on generic driver support in nRF Connect SDK come with a decision on the placement on !BT in the driver

Imply on Zigbee was discussed in a call and other options. There are surely multiple ways to solve this, but the additions in this PR was cleanly stated as a **temporary fix** to try to get CI green.

We will address this issue later on when we have more time,  but probably in a different PR than this

I will remember to link to this, soon to be closed PR in any follow-up discussions